chapter 2.1 - Make Bastion working (#8)
bastion-rs/book: src/definition-01.md

> # Definition

Reviewer: ```suggestion
# Definitions
```

> Quick reminder to some notion that you will encounter.

Reviewer: ```suggestion
Quick explanation of some words you will encounter.
```
